guides/introduction/quick_start_guide.md

```diff
@@ -114,3 +114,180 @@ This will lead you to a web page with a search bar that will allow you to search
+    <script src="https://cdn.jsdelivr.net/npm/instantsearch.js@4"></script>
+    <script>
+        const search = instantsearch({
+            indexName: "movies",
+            searchClient: instantMeiliSearch(
+                "http://localhost:7700"
+            )
+            });
+
+            search.addWidgets([
+            instantsearch.widgets.searchBox({
+                container: "#searchbox"
+            }),
+            instantsearch.widgets.hits({
+                container: "#hits",
+                templates: {
+                item: `
+                    <div>
+                    <div class="hit-name">
+                        {{#helpers.highlight}}{ "attribute": "title" }{{/helpers.highlight}}
+                    </div>
+                    </div>
+                `
+                }
+            })
+            ]);
+
+            search.start();
+    </script>
+  </body>
+</html>
+```
+
+The code above comes in multiple parts:
+
+- The body part of the page adds both `searchbox` and `hits` elements.
+  `instant-meilisearch` adds the search bar and search results in these elements.
+- `<scripts src="..">` tags are [CDN](https://en.wikipedia.org/wiki/Content_delivery_network)'s that imports libraries needed to run `instant-meilisearch`.
+- The JavaScript part is where you customize `instant-meilisearch`.
+
+To use `instant-meilisearch` using `npm` or `yarn` please visit [instant-meilisearch](https://github.com/meilisearch/instant-meilisearch).
+
+:::
+
+::: tab Vue.js
+
+The following code sample uses [Vue.js](https://vuejs.org/) framework.
+
+```html
+<!DOCTYPE html>
+<html>
+  <body>
+    <div id="app">
+      <ais-instant-search :search-client="searchClient" index-name="movies" >
+        <ais-search-box placeholder="Search here…" class="searchbox"></ais-search-box>
+        <ais-hits>
+          <div slot="item" slot-scope="{ item }">
+            <ais-highlight :hit="item" attribute="title" />
+          </div>
+        </ais-hits>
+      </ais-instant-search>
+    </div>
+  </body>
+  <script src="https://cdn.jsdelivr.net/npm/vue"></script>
+  <script src="https://cdn.jsdelivr.net/npm/vue-instantsearch@3.2.0/dist/vue-instantsearch.js"></script>
+  <script src="https://cdn.jsdelivr.net/npm/@meilisearch/instant-meilisearch/dist/instant-meilisearch.umd.min.js"></script>
+  <script>
+    Vue.use(VueInstantSearch)
+    var app = new Vue({
+        el: '#app',
+        data: {
+            searchClient: instantMeiliSearch('http://127.0.0.1:7700')
+        }
+    })
+  </script>
+</html>
+```
+
+The code above comes in multiple parts:
+
+- The body part adds the `Vue.js` components.
+  `instant-meilisearch` adds the search bar and search results in these components.
+- `<scripts src="..">` tags are [CDN](https://en.wikipedia.org/wiki/Content_delivery_network)'s that import libraries needed to run `instant-meilisearch`.
+- The `Vue.js` part is where you customize your `instant-meilisearch`.
```

Comment:
```
- The body part adds the `Vue.js` components.
  `instant-meilisearch` adds the search bar and search results in these components.
- `<scripts src="..">` tags are [CDN](https://en.wikipedia.org/wiki/Content_delivery_network)'s that import libraries needed to run `instant-meilisearch`.
- The `Vue.js` part is where you customize your `instant-meilisearch`.
```
This explanation seems vague to me. Maybe someone more familiar with Vue could clarify the correct way to explain what the above code is doing?

Reply:
I made changes to the explaination! What do you think of it? @react-learner